src/main/java/qna/domain/Answer.java

```diff
@@ -86,8 +88,16 @@ public boolean isDeleted() {
         return deleted;
     }
 
-    public void delete() {
+    public DeleteHistory delete(User loginUser) throws CannotDeleteException {
+        validateIsOwner(loginUser);
         this.deleted = true;
+        return new DeleteHistory(ContentType.ANSWER, this.id, this.writer, LocalDateTime.now());
+    }
+
+    private void validateIsOwner(User loginUser) throws CannotDeleteException {
+        if (!isOwner(loginUser)) {
```

Comment:
```suggestion
        if (isNotOwner(loginUser)) {
```
이렇게 써보는건 어떨까요? :)
메서드 네이밍 관련해서 [참고](https://velog.io/@hellojihyoung/%EA%B0%9C%EB%B0%9C-%EB%8F%84%EC%84%9C-Clean-Code-17%EC%9E%A5-%EB%83%84%EC%83%88%EC%99%80-%ED%9C%B4%EB%A6%AC%EC%8A%A4%ED%8B%B1#-g29-%EB%B6%80%EC%A0%95-%EC%A1%B0%EA%B1%B4%EC%9D%80-%ED%94%BC%ED%95%98%EB%9D%BC:~:text=%EB%B6%80%EC%A0%95%20%EC%A1%B0%EA%B1%B4%EC%9D%80%20%EA%B8%8D%EC%A0%95%20%EC%A1%B0%EA%B1%B4%EB%B3%B4%EB%8B%A4%20%EC%9D%B4%ED%95%B4%ED%95%98%EA%B8%B0%20%EC%96%B4%EB%A0%B5%EB%8B%A4.%20%EA%B0%80%EB%8A%A5%ED%95%98%EB%A9%B4%20%EA%B8%8D%EC%A0%95%20%EC%A1%B0%EA%B1%B4%EC%9D%84%20%ED%91%9C%ED%98%84%ED%95%9C%EB%8B%A4.) 해보시면 좋을 것 같아요~!

Reply:
`부정 조건은 긍정 조건보다 이해하기 어렵다. 가능하면 긍정 조건을 표현한다.`

확실히 잘 보이지도 않고 헷갈리는 `!`를 읽는 것보단 부정 메서드를 만드는 것이 좋을 것 같습니다. 또 람다식일 경우 축약도 가능해져서 더 좋은것 같습니다ㅎ